packages/contracts-bedrock/test/legacy/L1BlockNumber.t.sol

```diff
@@ -4,24 +4,40 @@ pragma solidity 0.8.15;
 // Testing
 import { Test } from "forge-std/Test.sol";
 
-// Contracts
-import { L1BlockNumber } from "src/legacy/L1BlockNumber.sol";
-import { L1Block } from "src/L2/L1Block.sol";
+// Scripts
+import { DeployUtils } from "scripts/libraries/DeployUtils.sol";
 
 // Libraries
 import { Predeploys } from "src/libraries/Predeploys.sol";
 
+// Interfaces
+import { IL1BlockNumber } from "src/legacy/interfaces/IL1BlockNumber.sol";
+import { IL1Block } from "src/L2/interfaces/IL1Block.sol";
+
 contract L1BlockNumberTest is Test {
-    L1Block lb;
-    L1BlockNumber bn;
+    IL1Block lb;
+    IL1BlockNumber bn;
 
     uint64 constant number = 99;
 
     /// @dev Sets up the test suite.
     function setUp() external {
-        vm.etch(Predeploys.L1_BLOCK_ATTRIBUTES, address(new L1Block()).code);
-        lb = L1Block(Predeploys.L1_BLOCK_ATTRIBUTES);
-        bn = new L1BlockNumber();
+        vm.etch(
+            Predeploys.L1_BLOCK_ATTRIBUTES,
+            address(
+                DeployUtils.create1({
```

Comment:
Can we just use `vm.getCode`?

Reply:
Would need to be `vm.getDeployedCode` but I prefer to do an explicit deploy if we need the deployed code to keep it consistent.

Reply:
If there is no constructor, there is no need to deploy. We are moving away from any predeploy having a constructor/initializer with standard L2 genesis so it will be an assumption that is consistent

Reply:
Mmm ok sounds good

Reply:
Updated